code/game/turfs/simulated/floor/planet_floor.dm

```diff
@@ -104,6 +105,7 @@
 	name = "ungenerated turf"
 	desc = "If you see this, and you're not a ghost, yell at coders"
 	environment_type = "genturf"
+	//initial_gas_mix = "o2=22;n2=82;TEMP=293.15"
```

Comment:
why u adding commented code

Reply:
Looks like a duplicate of line 4. Likely debug code that they forgot to remove? Seems like it can be safely removed